version/version.go

```diff
@@ -8,7 +8,7 @@ var Package = "github.com/distribution/distribution/v3"
 // the latest release tag by hand, always suffixed by "+unknown". During
 // build, it will be replaced by the actual version. The value here will be
 // used if the registry is run after a go get based install.
-var Version = "v3.0.0+unknown"
+var Version = "v3.0.0-alpha.1"
```

Comment:
do we need a second commit in this PR to move it back to `+unknown` (to prevent new builds from master from showing `v3.0.0-alpha.1`)?

Reply:
why do we want to prevent that? We actually want that `alpha` suffix to show in builds 🤔 am I missing something here?

Reply:
I think we only want that when it's built from the tag itself, and not for commits on `main` _after that_ (otherwise it may look like they were the `-alpha.1` release, but they're .. not actually

Reply:
We can change it back to `+unknown` immediately after it's merged though, but we probably shouldn't forget (hence; it could be in the same PR, but then tag the first commit, not the last one; not sure if the release tool supports that)

Reply:
> I think we only want that when it's built from the tag itself, and not for commits on main after that (otherwise it may look like they were the -alpha.1 release, but they're .. not actually

This is...complicated....the way these tools version their stuff is. I never liked that. It's cute, but no. Anything on `main` that's been built off alpha release should have `v3.0.0.-alpha.1-COMMIT-SHA` IMHO.